test/test_AutoindexHandler.cpp

```diff
@@ -0,0 +1,38 @@
+#include "gmock/gmock.h"
```

Comment:
The compiler can't find the gmock header.

Reply:
You need to rebuild your devcontainer